vic/vic_run/src/lakes.eb.c

```diff
@@ -290,15 +290,16 @@ solve_lake(double           snowfall,
         *  6. Calculate initial energy balance over ice.
         **********************************************************************/
 
+        windi = (wind * log((2. + soil_con.snow_rough) / soil_con.snow_rough) /
+                 log(wind_h / soil_con.snow_rough));
+        if (windi < 1.0) {
+            windi = 1.0;
+        }
+
```

Comment:
Personally I'd vote for leaving the `windi` calculation inside the `if` statement, and then having `windi` set to 0 in the `else` block that begins at line 384.  This would keep it consistent with the other lake-ice-related terms.